.gitignore

```diff
@@ -4,6 +4,7 @@
 *.so
 *.service
 venv
+.venv
```

Comment:
um idk if i wanna add this one to gitignore seems like a drastic change 🤔

Reply:
.venv is another common name for virtual env folder. it's not used for anything else

Reply:
propaganda

Reply:
if you dont merge this im sending you to gulag